tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -75,6 +75,11 @@ trait Sharing {
 	 */
 	private array $createdPublicShares = [];
 
+	/**
+	 * @var array
+	 */
+	private array $shareNGCreatedPublicSharesCreated = [];
```

Comment:
maybe we can use `shareNg` everywhere
```suggestion
	private array $shareNgCreatedPublicSharesCreated = [];
```